src/app/ui/contentfortabworkplace.h

```diff
@@ -0,0 +1,43 @@
+#ifndef CONTENTFORTABWORKPLACE_H
+#define CONTENTFORTABWORKPLACE_H
+
```

Comment:
http://stackoverflow.com/questions/4757565/c-forward-declaration
HistoryTree *uiHistoryTree;
+    RevisionView *uiHistoryTreeContents;
+    EditorView *uiEditorView;
+    FileView *uiFileViewContents;
+    DockFileView *uiFileView;
+    RevisionTable *uiRevisionTable;
+    DockConsole *uiConsole;
+    ConsoleView *uiConsoleContents;
that`s just pointers